engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ConstraintValidatorDescriptor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.engine.constraintvalidation;
+
+import java.io.Serializable;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.EnumSet;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorFactory;
+import javax.validation.constraintvalidation.ValidationTarget;
+
+import org.hibernate.validator.cfg.context.ConstraintDefinitionContext.ValidationCallable;
+
+/**
+ * Represents a specific validator type.
```

Comment:
Not sure the Javadoc is accurate (see my remark about `validatorTypes` earlier).